src/graphql/query/userAchievementProgress.graphql

```diff
@@ -1,9 +1,19 @@
 query UserAchievementProgress {
   userAchievementProgress {
     id
+    lendingAchievements {
```

Comment:
These two queries were combined, but I only see locations where we use either `lendingAchievements` or `tieredLendingAchievements`. Do we need this combined version or separate queries?

Reply:
This will be helpful for this my kiva ticket: https://kiva.atlassian.net/browse/MP-951

Reply:
Okay that makes sense. I hadn't seen that ticket before, but I see that it's a new ticket.